Doc/requirements.txt

```diff
@@ -7,7 +7,7 @@ sphinx==4.5.0
 
 blurb
 
-sphinx-lint<1
+sphinx-lint<1,!=0.6.2
```

Comment:
For future us, it would be useful to add a comment with reasoning behind `!=0.6.2`.